web/auth/deprecated_app_list.go

```diff
@@ -85,7 +86,11 @@ func (d *DeprecatedAppList) RenderArgs(client *oauth.Client, inst *instance.Inst
 		"Favicon":     middlewares.Favicon(inst),
 		"AppName":     app.Name,
 		"Platform":    platform,
-		"StoreURL":    storeURL,
+		// template.URL is used in order to avoid the discard of url starting
+		// by `market://` (the url is replaced by "#ZgotmplZ" in case of discard).
+		//
+		// More details at https://github.com/golang/go/blob/master/src/html/template/url.go#L19
```

Comment:
a stable url seems better to avoid broken url while code changes

https://github.com/golang/go/blob/bce7aec3cdca8580585095007e9b7cea11a8812f/src/html/template/url.go#L19